ARGO/src/TransformComponent.cpp

```diff
@@ -12,7 +12,9 @@ TransformComponent::TransformComponent(float t_x, float t_y, float t_rotation) :
 {
 	m_pos.x = t_x;
 	m_pos.y = t_y;
-	m_rotation = t_rotation;
+	m_rotation = t_rotation; 
```

Comment:
will you through in the comment from physics system that explains the minus 1 for left and right rotation or make a utilities value for it like int INVALID_ROTATION = -1

Reply:
What I'm wondering though is why you have 2 variables for this, why are you not using 1 variable where negative is one direction and positive is the other and 0 is neither?